[SN-538] docs: fix feed manager documentation (#1)
LinioIT/seller-center-sdk: docs/Managers/Feed.md

## Getting the feed

> It's possible to get a specific feed specifying it's an identifier to the method ```getFeedStatusById```

Reviewer: ```suggestion
It's possible to get a specific feed by providing the identifier into the  ```getFeedStatusById``` method.
```
